src/WSModel.php

```diff
@@ -1,6 +1,6 @@
 <?php
 //**********************************************************************************************
-//                               WSModel.php 
+//                               WSModel.php
```

Comment:
why did you commited this file ? No code modification has been done here..

Reply:
The modification have been done by PHP CS Fixer . It finds an extra space after the filename and remove it.

Reply:
ok for the spaces but not for : 
- the lines added 
- the function(arg)
{
} 

it does not follows the OpenSILEX dev rules